### PR TITLE
added the best kind of emojis

### PR DESCRIPTION
**Pull request by delaynieym**
todo: should later add a list of the emoji commands

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -133,7 +133,7 @@ impl EventHandler for Handler {
 }
 
 #[group]
-#[commands(multiply, ping, quit, dia)]
+#[commands(multiply, ping, quit, dia, owo, uwu, smile, hug, flex, animal, surprise, dance, shrug, flip, unflip, sus, cri, yike, bear)]
```
Comment (captainrex493) [inline src/main.rs:136]: Move the emoji commands into a different group

Comment (delaynieym) [reply src/main.rs:136]: how?

Comment (captainrex493) [reply src/main.rs:136]: add the following beneath the first group
#[group]
#[commands(ALL OF YOUR COMMANDS)]

Comment (captainrex493) [reply src/main.rs:136]: https://docs.rs/serenity/0.8.6/serenity/framework/standard/macros/attr.group.html

Comment (delaynieym) [reply src/main.rs:136]: do i need to do something here too? 
image

Comment (captainrex493) [reply src/main.rs:136]: image
image
Something like that

Comment (captainrex493) [reply src/main.rs:136]: Except emoji

**Pull request merged**